charts/hedera-json-rpc-relay-websocket/templates/configmap.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "json-rpc-relay-websocket.fullname" . }}
+  labels:
+    app:  {{ template "json-rpc-relay-websocket.name" . }}
+    {{ include "json-rpc-relay-websocket.labels" . | nindent 4 }}
+data:
```

Comment:
A lot of configs below are not utilized in the websocket server package.
Can you reduce this to only the ones utilized by the websocket server as the default configuration